sidecar/Dockerfile.ubi

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.16.2 as go-builder
+RUN mkdir /build
+ADD ./out_gstdout /build/
+WORKDIR /build
+RUN make all
+
+FROM public.ecr.aws/sumologic/fluent-bit:1.6.10-ubi
+
+ARG VERSION=${VERSION}
+ARG RELEASE_NUMBER=${RELEASE_NUMBER}
+
+LABEL name="Tailing Sidecar" \
+      maintainer="collection@sumologic.com" \
+      vendor="sumologic" \
```

Comment:
```suggestion
      vendor="SumoLogic" \
```

?

Reply:
I think that we should write company name as "Sumo Logic". What do you think?

Reply:
🤔  might be. We'd have to change the labels in other images as well